plugins/base/src/main/kotlin/translators/psi/DefaultPsiToDocumentableTranslator.kt

```diff
@@ -250,7 +257,7 @@ object DefaultPsiToDocumentableTranslator : SourceToDocumentableTranslator {
                     DParameter(
                         dri.copy(target = dri.target.nextTarget()),
                         psiParameter.name,
-                        javadocParser.parseDocumentation(psiParameter).toPlatformDependant(),
+                        DocumentationNode(docs.entries.mapNotNull { it.value.children.filterIsInstance<Param>().firstOrNull { it.root.children.firstIsInstanceOrNull<DocumentationLink>()?.children?.firstIsInstanceOrNull<Text>()?.body == psiParameter.name } }).toPlatformDependant(),
```

Comment:
Well, you first wrap DocumentationNode with Map to just a moment later unwrap it...
You can go `val docs = javadocParser.parseDocumentation(psi) `
And I suggest creating something like this:
```
public interface WithChildren {
    val children: List<*>
}

public inline fun <reified T> WithChildren.firstChildOfType() =
    children.filterIsInstance<T>().firstOrNull()

public inline fun <reified T> WithChildren.firstChildOfType(predicate: (T) -> Boolean) =
    children.filterIsInstance<T>().firstOrNull(predicate)
```

and adding it to DocumentationNode, DocTag and TagWrapper. Then this invocation will look like this:

```
DocumentationNode(
    listOfNotNull(docs.firstChildOfType<Param> {
        it.firstChildOfType<DocumentationLink>()?.firstChildOfType<Text>()?.body == psiParameter.name
     })
).toPlatformDependant()
```